Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamerMSE.cpp

```diff
@@ -77,17 +77,25 @@ using namespace std;
 
 namespace WebCore {
 
+struct PadProbeInformation
+{
+    AppendPipeline* m_appendPipeline;
+    const char* m_description;
+    gulong m_probeId;
+};
+
 class AppendPipeline : public ThreadSafeRefCounted<AppendPipeline> {
 public:
     enum AppendStage { Invalid, NotStarted, Ongoing, KeyNegotiation, DataStarve, Sampling, LastSample, Aborting };
 
-    static const unsigned int s_dataStarvedTimeoutMsec = 1000;
```

Comment:
Yes and no. This case is triggered only in cases happening not very often at all so it is not a performance loss and you also told me that having these low timeouts could cause issues in Debug mode, etc. Because of these reasons I increased the value and I think 2s is ok for the cases it has to be triggered.